package.json

```diff
@@ -29,6 +29,9 @@
   },
   "homepage": "https://github.com/mixmaxhq/eslint-config-mixmax#readme",
   "peerDependencies": {
-    "eslint": ">= 3"
-  }
+    "babel-eslint": "^7.2.3",
+    "eslint": ">= 3",
+    "eslint-plugin-react": "^7.1.0"
```

Comment:
we can make it so that projects that use this module don't need to include `eslint`, `babel-eslint`, etc. in their `devDependencies`, but doing so entails requiring engineers to install those packages globally. since that's a bit of a hassle, i think we should include those modules in parent projects' `devDependencies`.

i'll update the README 👍